app/js/helper_profile_controller.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+/* global SumoDB, nunjucks  */
+
+(function(exports) {
+  var MSG_NO_QUESTIONS = 'No questions found';
+  var PROFILE_DETAILS_TMPL = 'helper_profile.html';
+  var QUESTION_LIST_TMPL = 'question_list_day.html';
+
+  var username;
+  var headline;
+  var profile_details_container;
+  var questions_container;
+
+  function load_profile() {
+    var promise = SumoDB.get_user(username, false, 96);
```

Comment:
`false` ? `96` ? This is not very readable. After looking at `SumoDB`, I think we should do a `get_public_user` function rather than different arguments.